Makefile

```diff
@@ -45,6 +45,20 @@ images: package/.image.shipyard-dapper-base
 .DEFAULT_GOAL := lint
 # Shipyard-specific ends
 
+# This removes all Submariner-provided images and all untagged images
+# Use this to ensure you use current images
+prune-subm-images:
+	docker images | grep -E '(admiral|lighthouse|nettest|shipyard|submariner|<none>)' | while read image tag hash _; do \
+	    if [ "$$tag" != "<none>" ]; then \
+	        docker rmi $$image:$$tag; \
```

Comment:
Why not just o something like `docker rmi $$hash || :` all the time?

Reply:
Right, we might as well just use the hash. We don’t need `|| :` here.

Reply:
Oh so I ran into the reason I did this originally: images can’t be deleted by reference if multiple tags point to them. So tagged images need to be deleted by tag first (and the image is deleted once the last tag referencing it is deleted).